index.bs

```diff
@@ -2073,6 +2151,69 @@ Methods</h4>
 		<div>
 			<em>Return type:</em> {{Promise}}&lt;{{undefined}}&gt;
 		</div>
+
+		: <dfn>setSinkId((DOMString or AudioContextSinkCategory) sinkId)</dfn>
+		::
+			Sets the ID of the output device if it is permitted to play audio. When
+			this method is invoked, the user agent MUST run the following steps:
+
+			<div algorithm="AudioContext.setSinkId()">
+				1. Let |document| be the current settings object's
+					[=associated Document=].
+
+				1. If |document| is not allowed to use the feature identified by
+					<code>"speaker-selection"</code>, return [=a promise rejected with=] a
+					new {{DOMException}} whose name is "{{NotAllowedError}}".
+
+				1. Let |context| be the {{AudioContext}} object on which this method was
+					invoked.
+
+				1. Let |sinkId| be the method's first argument.
+
+				1. If |sinkId| is equal to |context|'s {{AudioContext/[[sink ID]]}}
+					internal slot, return a promise resolved with {{undefined}}.
+
+				1. Let |p| be a new promise.
+
+				1. Run the following substeps:
+
+					1. If |sinkId| is the empty string, reject |p| with a new
```

Comment:
If the context is already closed, we should reject the promise as well.

Reply:
Agreed.